sys/arduino/Makefile.include

```diff
@@ -4,14 +4,14 @@ SKETCHES = $(wildcard $(APPDIR)/*.sketch)
 
 ifneq (,$(SKETCHES))
   # Define application sketches module, it will be generated into $(BINDIR)
-  SKETCH_MODULE     ?= arduino_sketches
   SKETCH_MODULE_DIR ?= $(BINDIR)/$(SKETCH_MODULE)
   include $(RIOTBASE)/sys/arduino/sketches.inc.mk
 
   # Depends on module
-  USEMODULE += $(SKETCH_MODULE)
   DIRS      += $(SKETCH_MODULE_DIR)
   BUILDDEPS += $(SKETCH_GENERATED_FILES)
+else
+  PSEUDOMODULES += $(SKETCH_MODULE)
```

Comment:
why it this needed ?

Reply:
```diff
diff --git a/sys/arduino/Makefile.include b/sys/arduino/Makefile.include
index 5e80e8e2ad..a460bbac13 100644
--- a/sys/arduino/Makefile.include
+++ b/sys/arduino/Makefile.include
@@ -10,8 +10,6 @@ ifneq (,$(SKETCHES))
   # Depends on module
   DIRS      += $(SKETCH_MODULE_DIR)
   BUILDDEPS += $(SKETCH_GENERATED_FILES)
-else
-  PSEUDOMODULES += $(SKETCH_MODULE)
 endif
 
 # include the Arduino headers

```

```
USEMODULE=arduino BOARD=nucleo-l152re make -C examples/hello-world/
make: Entering directory '/home/francisco/workspace/RIOT/examples/hello-world'
Building application "hello-world" for "nucleo-l152re" with MCU "stm32".

"make" -C /home/francisco/workspace/RIOT/boards/nucleo-l152re
"make" -C /home/francisco/workspace/RIOT/boards/common/nucleo
"make" -C /home/francisco/workspace/RIOT/core
"make" -C /home/francisco/workspace/RIOT/cpu/stm32
"make" -C /home/francisco/workspace/RIOT/cpu/cortexm_common
"make" -C /home/francisco/workspace/RIOT/cpu/cortexm_common/periph
"make" -C /home/francisco/workspace/RIOT/cpu/stm32/bootloader
"make" -C /home/francisco/workspace/RIOT/cpu/stm32/periph
"make" -C /home/francisco/workspace/RIOT/cpu/stm32/stmclk
"make" -C /home/francisco/workspace/RIOT/cpu/stm32/vectors
"make" -C /home/francisco/workspace/RIOT/drivers
"make" -C /home/francisco/workspace/RIOT/drivers/periph_common
"make" -C /home/francisco/workspace/RIOT/sys
"make" -C /home/francisco/workspace/RIOT/sys/arduino
"make" -C /home/francisco/workspace/RIOT/sys/auto_init
"make" -C /home/francisco/workspace/RIOT/sys/div
"make" -C /home/francisco/workspace/RIOT/sys/newlib_syscalls_default
"make" -C /home/francisco/workspace/RIOT/sys/pm_layered
"make" -C /home/francisco/workspace/RIOT/sys/stdio_uart
"make" -C /home/francisco/workspace/RIOT/sys/xtimer
arm-none-eabi-gcc: error: /home/francisco/workspace/RIOT/examples/hello-world/bin/nucleo-l152re/arduino_sketches.a: No such file or direc
```

Reply:
I want to keep the behaviour as it was before, so no need to manually include the sketch module, if there are any it works, if there are none, it works.